wsd/ClientSession.cpp

```diff
@@ -690,6 +690,11 @@ bool ClientSession::_handleInput(const char *buffer, int length)
                                 << "] has no write permissions in Storage and cannot save.");
             sendTextFrameAndLogError("error: cmd=save kind=savefailed");
         }
+        else if(_wopiFileInfo && _wopiFileInfo->getHideSaveOption())
+        {
+            LOG_WRN("Session [" << getId() << "] on document [" << docBroker->getDocKey()
```

Comment:
I assume the UI will have blocked the save as well. So this warning is to signal that someone is trying to hack the UI?

Reply:
Yes